cpp/test/CMakeLists.txt

```diff
@@ -54,7 +54,8 @@ if(BUILD_CUML_TESTS)
     sg/trustworthiness_test.cu
     sg/tsne_test.cu
     sg/tsvd_test.cu
-    sg/umap_test.cu)
+    #sg/umap_test.cu
```

Comment:
Remove commented out code. If this test has been fully replaced and is no longer needed, let's remove it altogether (assuming the new one already covers these tests).